lib/rules/space-infix-ops.js

```diff
@@ -48,7 +48,28 @@ module.exports = function(context) {
      * @private
      */
     function report(mainNode, culpritToken) {
-        context.report(mainNode, culpritToken.loc.start, "Infix operators must be spaced.");
+        context.report({
+            node: mainNode,
+            loc: culpritToken.loc.start,
+            message: "Infix operators must be spaced.",
+            fix: function(fixer) {
+                var previousToken = context.getTokensBefore(culpritToken, 1);
+                var afterToken = context.getTokensAfter(culpritToken, 1);
+                var fixString = "";
+
+                if (culpritToken.range[0] - previousToken[0].range[1] === 0) {
+                    fixString = " ";
```

Comment:
I know we’ve already implemented autofixing for some whitespace rules, but I wonder if a whitespace is always correct here, it could be also a linebreak, right? Another problem would be that this fix could possible create errors for the `max-len` rule.